util/kustomize/kustomize.go

```diff
@@ -72,7 +72,29 @@ type kustomize struct {
 	noProxy string
 }
 
-var _ Kustomize = &kustomize{}
+var KustomizationNames = []string{"kustomization.yaml", "kustomization.yml", "Kustomization"}
+
+// IsKustomization checks if the given file name matches any known kustomization file names.
+func IsKustomization(path string) bool {
+	for _, kustomization := range KustomizationNames {
+		if path == kustomization {
+			return true
+		}
+	}
+	return false
+}
+
+// LookKustFile looks for any known kustomization file in the path
+func LookKustFile(dir string) string {
```

Comment:
Can these 3 things be private?

Reply:
Nevermind, I see that the first two were public. Can the new function be private? :-)

Reply:
changed.